operator/helper/parser.go

```diff
@@ -25,6 +26,7 @@ type ParserConfig struct {
 	ParseFrom            entry.Field           `json:"parse_from" yaml:"parse_from"`
 	ParseTo              entry.Field           `json:"parse_to"   yaml:"parse_to"`
 	Preserve             bool                  `json:"preserve"   yaml:"preserve"`
+	PreserveAt           *entry.Field          `json:"preserve_at" yaml:"preserve_at"`
```

Comment:
I'm fine with `PreserveAt`, but should we consider `PreserveTo` in order to keep the language a bit more aligned with `ParseTo`?

Also, if it makes sense to use a pointer here, should we also on `ParseFrom` and `ParseTo`?

Reply:
Yeah, `PreserveTo` makes sense to me.